lib/uniware/client.rb

```diff
@@ -176,6 +176,31 @@ def create_pending_purchase_order(data, code)
       perform_operation("CreatePurchaseOrderRequest", body, facility_endpoint("#{code}"))
     end
 
+    def get_sale_order(sale_order_code)
+      body = Gyoku.xml(namespaced_hash({"Code" => sale_order_code}))
+      body = "<ser:SaleOrder>%s</ser:SaleOrder>" % [body]
+      perform_operation("GetSaleOrderRequest", body)
+    end
+
+
+    def switch_sale_order_item_facility_request(data)
```

Comment:
Yes Rajat. It is same. I did not notice that. We will decline amrita's pull request as I have added another method as well. I will make the changes you commented in the other pull request.